framework/core/src/Forum/ValidateCustomLess.php

```diff
@@ -68,7 +68,10 @@ function ($settings) use ($event) {
         );
 
         $assetsDir = $this->assets->getAssetsDir();
-        $this->assets->setAssetsDir(new FilesystemAdapter(new Filesystem(new NullAdapter)));
+
+        $adaptor = new InMemoryFilesystemAdapter();
+        $filesystem = new Filesystem($adaptor);
+        $this->assets->setAssetsDir(new FilesystemAdapter($filesystem, $adaptor));
```

Comment:
```suggestion
        $adapter = new InMemoryFilesystemAdapter();
        $filesystem = new Filesystem($adapter);
        $this->assets->setAssetsDir(new FilesystemAdapter($filesystem, $adapter));
```

Reply:
Even though the naming has a typo. I would highly recommend to return to not using the temporary variables as they are not being used further down. Extra variables create unnecessary memory usage. It also doesn't really add any value specifying these out.

Ref: https://www.quora.com/Does-declaring-an-unnecessary-variable-in-PHP-consume-memory